src/nuklear_9slice.c

```diff
@@ -0,0 +1,106 @@
+#include "nuklear.h"
+#include "nuklear_internal.h"
+
+/* ===============================================================
+ *
+ *                          9-SLICE
+ *
+ * ===============================================================*/
+NK_API struct nk_9slice
+nk_sub9slice_ptr(void *ptr, nk_ushort w, nk_ushort h, struct nk_rect rgn, nk_ushort l, nk_ushort t, nk_ushort r, nk_ushort b)
+{
+    struct nk_9slice s;
+    nk_zero(&s, sizeof(s));
+    struct nk_image *i = (struct nk_image*)&s;
```

Comment:
Can we get this changed, @dav64 ? It makes the code less clear, as it is, IMO.
Other that this, the rest of the change looks good.
It would be good to have a usage example in the demos -- @FrostKiwi, are you still up for writing a new skinning example?

Reply:
After code investigating, I didn't found if there were specific reason for the original author (Michael) to not using something like @Hejsil solution as it also look cleaner to me
Maybe that only was Nuklear casting fashion at the time of writting :thinking: 
I'll update that

Reply:
Updated, let me know if you need anything else :slightly_smiling_face: